ProblemsSolution/ProblemsSolution.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Linq;
+
+namespace ProblemsSolution
+{
+    public class ArrayProblemsSolution
+    {
+        public static int SumArrayEvenNumbers(int[] numbers)
+        {
+            int sum = numbers.Where(number => number % 2 == 0).Aggregate((a, b) => a+b);
+            return sum;
+        }
+
+        public static int[] BigestTwoNumbersArray(int[] numbers)
+        {
+            if(numbers.Length == 0) return new int[] {};
+            if(numbers.Length == 1) return new int[] {numbers[0]};
+
+            int secondBiggest = Int32.MinValue;
+            int biggest = Int32.MinValue;
+
+            foreach(int number in numbers)
+            {
+                if(number > biggest)
+                {
+                    secondBiggest = biggest;
+                    biggest = number;
+                } else if(number > secondBiggest)
+                {
+                    secondBiggest = number;
+                }
+            }
+
+            return new int[] {secondBiggest, biggest};
+        }
+
+        public static bool DuplicatesInArray(int[] numbers)
+        {
+            int indexSum = 0;
+            int arraySum = 0;
+
+            for(int i=1; i <= numbers.Length; i++){
+                indexSum += i;
+                arraySum += numbers[i - 1];
+            }

```

Comment:
Not the right solution

It will fail for a simple array like: [2, 2, 2]

Reply:
Fixed: 3d9a558d4d79936dcaf2cd33653540b7dbe1d432 not sure if I should have made a Product extension method since it didn't feel needed.